Adding document for setting introduced : cluster.remote_store.translo… (#7078)
opensearch-project/documentation-website: _tuning-your-cluster/availability-and-recovery/remote-store/index.md

|:----------------------------------------------|:----------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| cluster.default.index.refresh_interval        | Time unit | Sets the refresh interval when the `index.refresh_interval` setting is not provided. This setting can be useful when you want to set a default refresh interval across all indexes in a cluster and also support the `searchIdle` setting. You cannot set the interval lower than the `cluster.minimum.index.refresh_interval` setting. |
| cluster.minimum.index.refresh_interval        | Time unit | Sets the minimum refresh interval and applies it to all indexes in the cluster. The `cluster.default.index.refresh_interval` setting should be higher than this setting's value. If, during index creation, the `index.refresh_interval` setting is lower than the minimum, index creation fails.                                       |

> | cluster.remote_store.translog.buffer_interval | Time unit | The default value of the translog buffer interval used when performing periodic translog updates. This setting is only effective when the index setting `index.remote_store.translog.buffer_interval` is not present.                                                                                                                   |

Reviewer: Can you undo the changes done in these lines?

Author: Done

> | cluster.remote_store.translog.max_readers     | Integer   | Sets the maximum number of open translog files for remote backed indexes. This setting is introduced to limit the total number of translog files per shard. Upon breaching which, flush will be triggered and translog files will be trimmed.                                                                                           |

Reviewer 2: ```suggestion
| cluster.remote_store.translog.max_readers     | Integer   | Sets the maximum number of open translog files for remote-backed indices. This setting limits the total number of translog files per shard. Upon breaching this limit, a flush will be triggered and the translog files will be trimmed. Default is `1000` and the minimum required is `100`.                                                                                          |
```

Added the default and minimum values enforced. Please adjust the above for formatting, if needed.

Author: Sure, added these changes.

| cluster.default.index.refresh_interval | Time unit | Sets the refresh interval when the `index.refresh_interval` setting is not provided. This setting can be useful when you want to set a default refresh interval across all indexes in a cluster and also support the `searchIdle` setting. You cannot set the interval lower than the `cluster.minimum.index.refresh_interval` setting. |
| cluster.minimum.index.refresh_interval | Time unit | Sets the minimum refresh interval and applies it to all indexes in the cluster. The `cluster.default.index.refresh_interval` setting should be higher than this setting's value. If, during index creation, the `index.refresh_interval` setting is lower than the minimum, index creation fails. |
| cluster.remote_store.translog.buffer_interval | Time unit | The default value of the translog buffer interval used when performing periodic translog updates. This setting is only effective when the index setting `index.remote_store.translog.buffer_interval` is not present. |

> | cluster.remote_store.translog.max_readers | Integer | Sets the maximum number of open translog files for remote-backed indexes. This limits the total number of translog files per shard. Upon breaching this limit, a flush will be triggered and the translog files will be trimmed. Default is `1000` and the minimum required is `100`.|

Reviewer 3: ```suggestion
| cluster.remote_store.translog.max_readers | Integer | Sets the maximum number of open translog files for remote-backed indexes. This limits the total number of translog files per shard. After reaching this limit, the remote-store flushes the translog files. Default is `1000`. The minimum required is `100`. |
```
